public/wp-content/themes/glific/functions.php

```diff
@@ -158,3 +158,44 @@ function show_more_blogs()
 if (function_exists('acf_add_options_page')) {
 	acf_add_options_page();
 }
+
+function wpb_move_comment_field_to_bottom($fields)
```

Comment:
@wevekamola is it a typo or there is some reason we are using `wpb`?

Reply:
@Satendra-SR Yes, it was supposed to be `wp`. Thanks for catching that. I have updated it